Gemfile

```diff
@@ -9,4 +9,5 @@ group :development, :test do
   gem 'gl_lint'
   gem 'rake', '~> 13.0'
   gem 'rspec', '~> 3.0'
+  gem 'rubocop-haml', '~> 0.2.4'
```

Comment:
This should actually be added to the `gl_rubocop.gemspec`

The Gemfile is only for development dependencies